Readme.md

```diff
@@ -1,13 +1,16 @@
-
 # Lombiq Walkthroughs for Orchard Core
 
 [![Lombiq.Walkthroughs NuGet](https://img.shields.io/nuget/v/Lombiq.Walkthroughs)](https://www.nuget.org/packages/Lombiq.Walkthroughs/) [![Lombiq.Walkthroughs.Test.UI NuGet](https://img.shields.io/nuget/v/Lombiq.Walkthroughs.Tests.UI)](https://www.nuget.org/packages/Lombiq.Walkthroughs.Tests.UI/)
 
 ## About
 
-Orchard Core Walkthroughs module for teaching Orchard Core fundamentals by guiding through the user in step by step guides.
+Do you want to learn about the most important [Orchard Core](https://orchardcore.net/) features with a hands-on tutorial that guides you with interactive tooltips, right there in Orchard Core? This module is for you! Check out a demo video of it [here](https://www.youtube.com/watch?v=mWfngBADvX0).
```

Comment:
You were faster than me! Since you've already started to link this, you can add this video to the root readme of OSOCE too. :)

Reply:
Ah, yes, thanks.